model-client/src/jvmMain/kotlin/org/modelix/model/oauth/ModelixOAuthClient.kt

```diff
@@ -34,7 +34,7 @@
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.withContext
 
-object ModelixOAuthClient {
+actual object ModelixOAuthClient {
```

Comment:
Can we add comments here to avoid further warnings? There's probably some assumptions hidden in the implementation in any case that could be described here.

Reply:
Done.
Added basic documenation.
More about assumptions is documented in `installAuth`.

Reply:
(Let see, if Dekekt is smart enaugh to notice that `expect object ModelixOAuthClient` is already documented.)

Reply:
Probably not ;-) You could just redirect the reader to the expect version.

Reply:
> You could just redirect the reader to the expect version.
```
@Suppress("UndocumentedPublicClass")  // already documented in the expected declaration
```
This comment redirects the user

Reply:
right